community_supplied/Kernel/Tnp/check-tnp-hello-drops-iagent.rule

```diff
@@ -0,0 +1,108 @@
+/*
+ * This rule monitors the health of the system with the help of the tnp hello drop count parameter. 
+ * 
+ * It uses iAgent sensor.
+ * The rule throws red alarm if the tnp hello drop count is above 100. 
+ * It throws yellow alarm if the tnp hello drop count is in the range 10-100.
+ * Else, the system is in a healthy condition.
+ */
+iceberg {
+    topic kernel.tnp {
+        rule check-tnp-hello-drops-iagent {
+            sensor tnp-hello-drop-sensor {
+                synopsis "tnp hello drop sensor";
+                description "iAgent sensor for TNP hello drops";
+                iAgent {
+                    file tnp_hello_drop_count.yml;
+                    table tnpHelloTable;
+                    frequency 1m;
+                }
+            }
+            field tnp-hello-drop-cnt {
+                sensor tnp-hello-drop-sensor {
+                    path tnp_hello_drop_cnt;
+                }
+                type integer;
+                description " TNP Hello Drop Count ";
+            }
+            trigger report-tnp-hello-drops-iagent {
+                synopsis "Trigger for TNP hello drops, with the sensor being Open config";
+                description "The above sensor helps in determining the health of the system based on the value of the TNP hello drop count. If this value is below 10, the system is in healthy condition. If the value is in the range 10-100, it is a cause for concern and must be taken as a warning. Anything above this, is a serious cause of concern and indicates that the system is under risk";
+                frequency 1m;
```

Comment:
Ensure trigger frequency and term time range is 1.5 times of sensor frequency in all the rules